stable/jenkins/templates/_helpers.tpl

```diff
@@ -275,7 +275,7 @@ Returns kubernetes pod template xml configuration
     <org.csanchez.jenkins.plugins.kubernetes.volumes.{{ $volume.type }}Volume>
   {{- end }}
   {{- range $key, $value := $volume }}{{- if not (eq $key "type") }}
-      <{{ $key }}>{{ $value }}</{{ $key }}>
+      <{{ $key }}>{{ if kindIs "string" $value }}{{ tpl $value $ | quote }}{{ else }}{{ $value }}{{ end -}}</{{ $key }}>
```

Comment:
the chomp in `{{ end -}}` is extraneous and can be removed

Reply:
Thank you @wmcdona89 . I have merged the latest code back and addressed these comments. 

Note: 
1. the latest change 1.23.1 did not seem to have a CHANGELOG entry when I merged
2. Had to rebase to sign an older commit

Reply:
@krishnakvall the 1.23.0 CHANGELOG entry is currently missing in this PR. can you please restore it and also add a 1.23.1 CHANGELOG entry. thanks!

```
1.23.1

Added auto detection for Ingress API Version
```

Reply:
Hi @wmcdona89 ,  have added 1.23.1 and fixed the removal of 1.23.0. Thank you!